tests/manifest.jsonld

```diff
@@ -10,6 +10,7 @@
     "flatten-manifest.jsonld",
     "fromRdf-manifest.jsonld",
     "remote-doc-manifest.jsonld",
-    "toRdf-manifest.jsonld"
+    "toRdf-manifest.jsonld",
+    "html-manifest.jsonld"
```

Comment:
This new manifest should probably also be mentioned in the _Design_ section of`tests/README.md`.